docs/release-notes/template.rst

```diff
@@ -0,0 +1,67 @@
+.. include:: ../references.txt
+
+.. _gammapy_{{ versiondata.version | replace('.', 'p') }}_release:
+
+{% set parts = versiondata.date.split('-') %}
+{% set year = parts[0] %}
+{% set month = parts[1]|int %}
+{% set day = parts[2]|int %}
+{% set month_name = ["January","February","March","April","May","June","July","August","September","October","November","December"][month-1] %}
+
+{% set suffix = "th" %}
+{% if day in [1,21,31] %}
+  {% set suffix = "st" %}
+{% elif day in [2,22] %}
+  {% set suffix = "nd" %}
+{% elif day in [3,23] %}
+  {% set suffix = "rd" %}
+{% endif %}
+
+
+{% if render_title %}
+{% if versiondata.name %}
+{% set title_text = versiondata.version ~ " (" ~ day|string ~ suffix ~ " " ~ month_name ~ ", " ~ year ~ ")" %}
+{{ title_text }}
+{{ top_underline * title_text|length }}
+{% endif %}
+{% endif %}
+
+- Released {{ day }}{{ suffix }} {{ month_name }}, {{ year }}
+- contributors
+- pull requests since v (not all listed below)
+- closed issues
+
+Summary
+-------
+
+
```

Comment:
If the summary template is completely defined from the beginning, is there a way to add something like a chapeau over the whole list of changes? e.g. what's new?